src/libstd/io/stdio.rs

```diff
@@ -872,6 +874,40 @@ impl fmt::Debug for StderrLock<'_> {
     }
 }
 
+/// Prints the given `str` and reads a [`String`] from
+/// [standard input](Stdin). The trailing newline is stripped.
+/// Gives an error on EOF (Ctrl-D).
+///
+/// # Note
+///
+/// If you require more explicit control over capturing
+/// user input, see the [`Stdin::read_line`] method.
+///
+/// # Examples
+///
+/// ```no_run
+/// #![feature(input)]
+/// use std::io::input;
+///
+/// let name = input("Enter name: ").expect("input failed");
+///
+/// println!("Your name is {}!", name);
```

Comment:
It's better to avoid unwrap() in examples.
```suggestion
/// #![feature(input)]
/// # fn main() -> std::io::Result<()> {
/// use std::io::input;
///
/// let name = input("Enter name: ")?;
///
/// println!("Your name is {}!", name);
/// # Ok(())
/// # }
```